### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by eranp22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,77 @@
...
+
+**Jumpcloud**
+| Parameter Name | Description | Required/Optional | Default |
+| --- | --- | --- | --- |
+| jumpcloud_api | A dictionary containing the JumpCloud API configurations. | Required | - |
+| url | The URL for the JumpCloud Insights Directory API endpoint. https://api.jumpcloud.com/insights/directory/v1/events. | Required | - |
+| start_date | The start date and time for querying the JumpCloud API in UTC time with the format of %Y-%m-%dT%H:%M:%S.%fZ. For example: 2023-05-04T12:30:00.000000Z. | Optional | The current date and time. Equals to nothing as Default |
+| credentials | A dictionary containing the token for authenticating the JumpCloud API request. | Required | - |
+| token | The JumpCloud API token. | Required | - |
+| time_interval | The time interval for querying the JumpCloud API in minutes. | Required | Equals to nothing as Default 5m |
```
Comment (ShiranAvidov) [inline README.md]: Set time_interval to Optional and in default write 5m

File: README.md [hunk 2]
```diff
@@ -1,2 +1,77 @@
...
+
+**`Config.yaml`**
+```yaml
+logzio:
+ url: "https://listener.logz.io:8071"
+ token: "Logzio_Token"
```
Comment (ShiranAvidov) [inline README.md]:
```suggestion
 token: "<<LOGZIO_TOKEN>>"
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,77 @@
...
+
+jumpcloud_api:
+   url: https://api.jumpcloud.com/insights/directory/v1/events
+   start_date:
+   credentials:
+     token: "jumpcloud_api_token"
```
Comment (ShiranAvidov) [inline README.md]:
```suggestion
     token: "<<JUMPCLOUD_API_TOKEN>>"
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,77 @@
 # logzio-jumpcloud
-Ship Jumpcloud logs to Logz.io
+Ship Jumpcloud logs to Logz.io. 
+
+Every time interval, events exported from Jumpcloud API is in the configuration and sends it to Logz.io.
```
Comment (ShiranAvidov) [inline README.md]:
```suggestion
Collects Jumpcloud events every time interval, using the Jumpcloud API given in the configuration, and sends them to Logz.io.
```

**Review by ShiranAvidov — CHANGES_REQUESTED** [review]

**Review by eranp22 — COMMENTED** [review]
README.md update

**Review by ShiranAvidov — APPROVED** [review]

**Comment by eranp22** [comment]
Create README.md

**Pull request merged**